frontend/src/App.svelte

```diff
@@ -164,13 +169,18 @@ Answer by running riddle.answer('your answer here') from the console.`);
           {:else}
             <ResourceTreeView
               rootResource="{rootResource}"
+              bind:bottomLeftPane="{bottomLeftPane}"
               bind:resourceNodeDataMap="{resourceNodeDataMap}"
               bind:modifierView="{modifierView}"
             />
           {/if}
         </Pane>
         <Pane slot="second" paddingVertical="{'1em'}">
-          <AttributesView resource="{currentResource}" />
+          {#if bottomLeftPane}
```

Comment:
It was not supposed to be a boolean based on what @marczalik and I discussed. It is supposed to be the actual "class" of component that is displayed in the bottom left pane (similar to `modifierView`). This if statement should check if that variable is set, or if it is `undefined` or `null`. Upon a second pass, it seems that it wasn't actually implemented this way.

When `bottomLeftPane` is implemented the same way as `modifierView`, the interfaces for `AttributesView` and `ScriptView` don't need to be interchangeable because `bottomLeftPane` will never have a value of `AttributesView`, it will only be `ScriptView`, `undefined`, or something that conforms to the same interface as `ScriptView`. 

For an example in the code base where there are panes that can optionally occupy the same space, but which have different interfaces, see the hex view/text view/disassembly view area.

https://github.com/redballoonsecurity/ofrak/blob/c08eebf3d7f99d16da0d0500373c237e84f19959/frontend/src/App.svelte#L181-L192

Reply:
Specifically, see this change in `ResourceTreeToolbar` that sets that bound variable:

https://github.com/redballoonsecurity/ofrak/blob/b2dda78d083b1800cb4003bc3011d18b550f60b9/frontend/src/ResourceTreeToolbar.svelte#L221

It should be exactly like the `modifierView` variable. In [this comment](https://github.com/redballoonsecurity/ofrak/pull/265/files/b2dda78d083b1800cb4003bc3011d18b550f60b9#r1147959815) I make a suggestion to change the code accordingly.

Reply:
One additional note is that (whether for `modifierView` or for `bottomLeftPane`), Svelte is very flexible about passing extra (or not enough) parameters to a component. The component will be instantiated and initialized just fine, it will just have some variables that are `undefined`. As long as the component checks this case (which it should anyway!) there won't actually be any problems.

That's a long way of saying that it doesn't actually matter if they implement the same interface. It will still work.

Reply:
Just a quick clarification here:
>  It is supposed to be the actual "class" of component that is displayed in the bottom left pane (similar to modifierView). This if statement should check if that variable is set, or if it is undefined or null. Upon a second pass, it seems that it wasn't actually implemented this way.
`bottomLeftPane` is not a boolean and it does contain the class of the component selected as seen here
https://github.com/redballoonsecurity/ofrak/blob/b2dda78d083b1800cb4003bc3011d18b550f60b9/frontend/src/ResourceTreeToolbar.svelte#L221